methods/iden3.json

```diff
@@ -0,0 +1,9 @@
+{
+  "name": "iden3",
+  "status": "registered",
+  "verifiableDataRegistry": "EVM compatible chains,
```

Comment:
I have a question.
Is the `"` at the end not necessary?

Reply:
```suggestion
  "verifiableDataRegistry": "EVM compatible chains",
```

Reply:
Good catch fixed it thanks!